src/decaf.rs

```diff
@@ -752,7 +757,7 @@ mod test {
     fn decaf_decompress_id() {
         let compressed_id = CompressedDecaf::identity();
         let id = compressed_id.decompress().unwrap();
-        assert_eq!(id.0.compress_edwards(), CompressedEdwardsY::identity());
+        assert_eq!(id.to_edwards().compress(), CompressedEdwardsY::identity());
```

Comment:
I think this should just be `0.compress()` since it's OK for the tests to use the internal structure.

Reply:
yep, also in d39cb275